apps/nrm/behaviours/save-form-session.js

```diff
@@ -53,7 +54,7 @@ module.exports = superclass => class extends superclass {
           return res.redirect('/nrm/save-and-exit');
         }
 
-        if (req.path.indexOf('/change') !== -1) {
+        if (req.path.match(/\/change$/)) {
```

Comment:
why do we have `edit` here https://github.com/UKHomeOffice/modern-slavery/pull/478/files#diff-c5f0f871388ee87ae3bd330abb55492cd697e4e358a49da73d8cbc5d117e4476R50

Reply:
Because we don't want steps ending in 'edit' or 'change' added to postgres as it causes frameworks issues. We only care about steps completed and that is it